aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{assert_success, harness::MoveHarness};
+use aptos_language_e2e_tests::account::Account;
+use aptos_types::{account_address::AccountAddress, transaction::SignedTransaction};
+use std::path::PathBuf;
+
+pub fn initialize(path: PathBuf) -> (MoveHarness, Account) {
+    let mut harness = MoveHarness::new();
```

Comment:
If we rebase on main, parallel execution is disabled in e2e tests by default. I have fixed that for the old aggregator file in main. But if have 2 initialisations this becomes error prone, so having a common APIs seems better to me, at least for code initialisation.

Reply:
I applied that parallel execution change to aggregator_v2 as well. The initialize functions in aggregator.vs and aggregator_v2.rs run different entry functions (0x1::aggregator_test::initialize, 0x1::aggregator_v2_test::initialize) and they use different feature flags. So, it would get messy to combine them.